packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts

```diff
@@ -7,54 +7,51 @@ import {<%= repositoryName %>} from '../repositories';
 export class <%= name %>Controller {
 
   constructor(
-    @inject('repositories.<%= modelNameCamel %>')
+    @inject('repositories.<%= repositoryName %>')
```

Comment:
Quick note: Was testing this on a real application and noticed that we must've changed our approach awhile ago, since the RepositoryMixin is now creating repositories with the name of the constructor object. Updated the template to reflect this so that it works out-of-the-box.